### ISSUE TITLE
gds.graph.export - Replace database if exists

### ISSUE THREAD

**Issue by Mintactus** [neo4j/graph-data-science-client#720]
Using
GDS Python client 1.11
Neo4j 5.22
GDS 2.8.0

As [gds.graph.export](https://neo4j.com/docs/graph-data-science/current/management-ops/graph-export/export-db/) does not support replacing an existing database, i have to create this process manueally, but it fails and I can't understand why? Anyway, it would be a way better workflow if a simple parameter like replace=True/False would exists for gds.graph.export. Also, it could simply create the database, why is the point to have to call CREATE DATABASE afterwhile ?

Simply dropping the database wasn't working properly either, so I tried to recreate and dump.

```
	def create_database_from_projection(self):
		projection = self.gds.graph.get(PROJECTION_NAME)
		database_config = {
			'dbName':CLIENT_NEO4J_DATABASE_NAME
		}
		self.gds.run_cypher(F"CREATE OR REPLACE DATABASE {CLIENT_NEO4J_DATABASE_NAME}")
		self.gds.run_cypher(F"DROP DATABASE {CLIENT_NEO4J_DATABASE_NAME} IF EXISTS")
		self.gds.graph.export(projection, **database_config)
		self.gds.run_cypher(F"CREATE DATABASE {CLIENT_NEO4J_DATABASE_NAME}")
```

I got this error:

neo4j.exceptions.ClientError: {code: Neo.ClientError.Procedure.ProcedureCallFailed} {message: Failed to invoke procedure `gds.graph.export`: Caused by: java.lang.IllegalArgumentException: The database [crotte] already exists. The graph export procedure can only create new databases.}

And yes...I named my database "crotte" :)

**Comment by knutwalker** [comment]
Hi @Mintactus,

The graph export procedure is basically a GDS version of [admin import](https://neo4j.com/docs/operations-manual/current/tools/neo4j-admin/neo4j-admin-import/#import-tool-full) -- it uses the same underlying API to get data into a new Neo4j Database. As such, it is also subject to the same restrictions, that is, it's writing the files for the database, but does otherwise assume that the database is offline and so we cannot interact with the DBMS, just with the files on disk. So, if the files for a database still exist, even after dropping it, we have to assume that the database exists, and when we have exported a graph, we only created the files, that's all we can do there. Since the importer is meant to be used with on offline database, but we offer an online alternative, we don't want to offer any way to potentially overwrite files of a live database.
